code/versioned/TestPageVersionedObject.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\Versioned\Model;
+
+use Page;
+use SilverStripe\Forms\GridField\GridField;
+use SilverStripe\Forms\GridField\GridFieldConfig_RecordEditor;
+
+/**
+ * Parent class of all test pages
+ */
+class TestPageVersionedObject extends Page implements \TestPageInterface
```

Comment:
```suggestion
class TestPageVersionedObject extends Page implements TestPageInterface
```

And add `use TestPageInterface;` above

Reply:
Updated